### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by 641i130** [cheat-engine/cheat-engine#1272]
How would one use this tool on Linux, if possible? Or how would one build this on Linux? I'm kind of surprised it has Mac support but no Linux support. Scanmem/Gameconquoer just isn't as good. I'm trying to find a 'speed hack' tool that changes increasing values but this one seems to be the only one that can since there aren't Linux versions.

**Comment by Symbai** [comment]
You need to run the server on Linux and the client on a Windows computer in the same network. Then attach the server to the game and the client to the server. Then use the client to debug your linux game.

**Comment by ftdomingo** [comment]
> You need to run the server on Linux and the client on a Windows computer in the same network. Then attach the server to the game and the client to the server. Then use the client to debug your linux game.
> 
> https://github.com/cheat-engine/cheat-engine/tree/master/Cheat%20Engine/ceserver

How do we go about building this one? is gcc enough?

**Comment by frostworx** [comment]
Hi,
no port, but maybe interesting:
With my yesterday's [release](https://github.com/frostworx/steamtinkerlaunch/releases/tag/v3.1.335) of [SteamTinkerLaunch](https://github.com/frostworx/steamtinkerlaunch) _("Linux wrapper tool for use with the Steam client for custom launch options and 3rd party programs")_ I added direct Cheat Engine [support](https://github.com/frostworx/steamtinkerlaunch#Cheat-Engine).
When enabled *(f.e. by simply putting the game into a "CheatEngine" Steam category)* Cheat Engine is downloaded and extracted *(if not already done)* and automatically started and stopped with the *(Proton)* game.

**Comment by fromtheeast710** [comment]
Though not perfect, running CheatEngine with Wine is a temporary alternative (will require _lib32-gnutls_, _libgnutls_ and _samba_).

**Comment by evpix** [comment]
Are there any changes to native Linux support?

**Comment by cheat-engine** [comment]
yes. you can download a native linux version from cheatengine.org

**Comment by D1mon** [comment]
@cheat-engine Finally. Thanks.
